src/main/kotlin/fi/metatavu/timebank/api/keycloak/KeycloakController.kt

```diff
@@ -67,6 +106,44 @@ class KeycloakController {
         }
     }
 
+    /**
+     * Updates Persons unspentVacations attribute
+     *
+     * @param user UserRepresentation
+     * @param newUnspent Int
+     * @return Int unspentVacations
+     */
+    fun updateUsersUnspentVacationDays(user: UserRepresentation, newUnspent: Int) {
+        val usersResource = getUsersResource()?.get(user.id)
+
+        try {
+            user.attributes["unspentVacations"] = listOf(newUnspent.toString())
+            usersResource?.update(user)
+        } catch (e: NullPointerException){
+            user.attributes = mapOf("unspentVacations" to listOf(newUnspent.toString()))
+            usersResource?.update(user)
```

Comment:
Is it certain that assigning attributes like this doesn't reset the other attributes of the user?